[2021-03-11 14:15:32,912] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: extract_standings.load_standings 2021-03-11T20:15:27.195758+00:00 [queued]>
[2021-03-11 14:15:32,918] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: extract_standings.load_standings 2021-03-11T20:15:27.195758+00:00 [queued]>
[2021-03-11 14:15:32,918] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-03-11 14:15:32,918] {taskinstance.py:1043} INFO - Starting attempt 1 of 3
[2021-03-11 14:15:32,918] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-03-11 14:15:32,929] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): load_standings> on 2021-03-11T20:15:27.195758+00:00
[2021-03-11 14:15:32,933] {standard_task_runner.py:52} INFO - Started process 58685 to run task
[2021-03-11 14:15:32,943] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'extract_standings', 'load_standings', '2021-03-11T20:15:27.195758+00:00', '--job-id', '176', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/standings_dag.py', '--cfg-path', '/var/folders/n1/tvvct6mn75xbwwwl4s02tfdc0000gn/T/tmpyzl7nlh3', '--error-file', '/var/folders/n1/tvvct6mn75xbwwwl4s02tfdc0000gn/T/tmplapge6sj']
[2021-03-11 14:15:32,946] {standard_task_runner.py:77} INFO - Job 176: Subtask load_standings
[2021-03-11 14:15:32,992] {logging_mixin.py:104} INFO - Running <TaskInstance: extract_standings.load_standings 2021-03-11T20:15:27.195758+00:00 [running]> on host 25.1.168.192.in-addr.arpa
[2021-03-11 14:15:33,032] {taskinstance.py:1255} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=AWTY
AIRFLOW_CTX_DAG_ID=extract_standings
AIRFLOW_CTX_TASK_ID=load_standings
AIRFLOW_CTX_EXECUTION_DATE=2021-03-11T20:15:27.195758+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-03-11T20:15:27.195758+00:00
[2021-03-11 14:15:33,033] {bash.py:135} INFO - Tmp dir root location: 
 /var/folders/n1/tvvct6mn75xbwwwl4s02tfdc0000gn/T
[2021-03-11 14:15:33,034] {bash.py:158} INFO - Running command: curl --header "Content-Type: application/json" --request POST --data @DatabaseServerContainer/standings_dump.json http://127.0.0.1:4321/db/update
[2021-03-11 14:15:33,045] {bash.py:169} INFO - Output:
[2021-03-11 14:15:33,064] {bash.py:173} INFO - Warning: Couldn't read data from file
[2021-03-11 14:15:33,064] {bash.py:173} INFO - Warning: "DatabaseServerContainer/standings_dump.json", this makes an empty
[2021-03-11 14:15:33,065] {bash.py:173} INFO - Warning: POST.
[2021-03-11 14:15:33,065] {bash.py:173} INFO -   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[2021-03-11 14:15:33,066] {bash.py:173} INFO -                                  Dload  Upload   Total   Spent    Left  Speed
[2021-03-11 14:15:33,070] {bash.py:173} INFO -   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0100   192  100   192    0     0  38400      0 --:--:-- --:--:-- --:--:-- 38400
[2021-03-11 14:15:33,071] {bash.py:173} INFO - <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
[2021-03-11 14:15:33,072] {bash.py:173} INFO - <title>400 Bad Request</title>
[2021-03-11 14:15:33,072] {bash.py:173} INFO - <h1>Bad Request</h1>
[2021-03-11 14:15:33,073] {bash.py:173} INFO - <p>The browser (or proxy) sent a request that this server could not understand.</p>
[2021-03-11 14:15:33,073] {bash.py:177} INFO - Command exited with return code 0
[2021-03-11 14:15:33,088] {taskinstance.py:1159} INFO - Marking task as SUCCESS. dag_id=extract_standings, task_id=load_standings, execution_date=20210311T201527, start_date=20210311T201532, end_date=20210311T201533
[2021-03-11 14:15:33,103] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-03-11 14:15:33,126] {local_task_job.py:146} INFO - Task exited with return code 0
